helpers/timesHelper.js

```diff
@@ -0,0 +1,167 @@
+const moment = require('moment-timezone');
+const logger = require('./logger');
+
+const DEFAULT_TIMEZONE = 'Europe/Berlin';
+
+let schoolTimezone;
+let userTimezone;
+let sameTimezone = true;
+
+/**
+ * @return {String} UTC offset as string based on current timezone, e.g +01:00
+ */
+const getUtcOffset = () => moment().format('Z');
+
+const getUserTimezone = (req) => (((req || {}).cookies || {}).USER_TIMZONE);
+
+/**
+ * @param {String} res result object containing shool data information
+ * sets default timezone if school timezone differs from the user timezone
+ */
+const setDefaultTimezone = (req, res) => {
+	schoolTimezone = (res.locals.currentSchoolData || {}).timezone;
+	userTimezone = getUserTimezone(req) || res.locals.currentTimezone;
+
+	if (schoolTimezone) {
+		res.locals.currentTimezone = schoolTimezone;
+	} else {
+		res.locals.currentTimezone = DEFAULT_TIMEZONE;
+	}
+	moment.tz.setDefault(res.locals.currentTimezone);
+	sameTimezone = res.locals.currentTimezone === userTimezone;
+
+	res.locals.currentTimezoneOffset = getUtcOffset();
+	res.locals.userTimezone = userTimezone;
+	res.locals.sameTimezone = sameTimezone;
+
+	logger.info(`timesHelper: instance timezone "${res.locals.currentTimezone} (${res.locals.currentTimezoneOffset})"`);
+	logger.info(`timesHelper: user timezone "${res.locals.userTimezone}"`);
+	logger.info(`timesHelper: same timezone "${JSON.stringify(res.locals.sameTimezone)}"`);
+};
+
+/**
+ * @param {Date} date UTC Date
+ * @return {moment} Date object based on current timezone
+ */
+const fromUTC = (date) => {
+	const result = moment(date);
+	logger.info(`timesHelper.fromUTC: ${date} to ${result.toISOString(true)}`);
+	return result;
+};
+
+/**
+ * @return {moment} Current date based on current timezone
+ */
+const currentDate = () => {
+	const result = moment();
+	logger.info(`timesHelper.currentDate: ${result.toISOString(true)}`);
+	return result;
+};
+
+/**
+ * @return {Number} Current time in seconds based on current timezone
+ */
+const now = () => {
+	const result = moment();
+	logger.info(`timesHelper.now: ${result}`);
+	return result.valueOf();
+};
+
+/**
+ *
+ * @param date
+ * @returns {string}
+ */
+const fromNow = (date) => moment(date).fromNow();
+
+/**
+ * @param {Date} date Date object
+ * @return {Object} Timestamp, date and time of given date as object
+ */
+const splitDate = (date) => {
```

Comment:
what is the format/type of date? for all exported functions i would like to have the format specified iso-string/Date-type or timestamp...

set default format/timezone from locale on load